applegacy/src/main/res/values-night/themes.xml

```diff
@@ -0,0 +1,18 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <!-- Base application theme. -->
+    <style name="Theme.Forzenbook" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
+        <item name="legacy_background">@color/lion_yellow</item>
+
+        <item name="legacy_primary">@color/black</item>
```

Comment:
camelCase, lots of comments from the other PR I think?

Reply:
I think I converted all snake case to camelcase, the only one that I did not was the colors which I left as for example dark_gray